srcpkgs/libbitcoin-blockchain/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'libbitcoin-blockchain'
+pkgname=libbitcoin-blockchain
+version=3.5.0
+revision=1
+archs="aarch64* ppc64* x86_64*"
```

Comment:
any particular reason why it is not available for arm-32bits ?

Reply:
libbitcoin-database requires 64-bit support, which would be revealed in 32-bit compilation error

it's been a while since i've attempted compiling on 32-bit, unfortunately no longer have notes on the exact error; suffice to say it makes 64-bit assumption

blockchain/node/server require db